### DISCUSSION TITLE
Pesquisa sobre a motivação das pessoas com contribuições em comunidades de programação

### DISCUSSION THREAD

**Discussion by woliveiras** [frontendbr/forum#1106] [Dicas]
Pessoal, estou fazendo uma pesquisa sobre a motivação das pessoas com contribuições em comunidades de programação. Tanto quem participa quanto quem não participa das comunidades, pode responder e irá ajudar muuuuito a melhorar nossos grupos.

https://goo.gl/forms/H6fJ1elWggTPj7K42

**Comment by caiomyrapereira** [comment]
como posso contribui  sendo nub? tipo é bem difícil kkk

**Comment by Cleysonlb** [comment]
Respondido @woliveiras 🎉

**Comment by phhoffmann** [comment]
ei @caiomyrapereira , saca só alguns links pra te ajudar:

[Contribuindo para projetos Open Source no Github mesmo sendo iniciante](https://woliveiras.com.br/posts/contribuindo-para-projetos-open-source-no-github-mesmo-sendo-iniciante/)

[Conheça algumas listas de projetos Open Source pra você contribuir desde o nível iniciante até avançado](https://medium.com/trainingcenter/saiba-como-contribuir-com-projetos-open-source-e-conhe%C3%A7a-o-ct-d916a30fcdb7)

**Comment by lidimayra** [comment]
@caiomyrapereira Tem um vídeo muuuuito legal da Emily Dunham sobre esse assunto. 5 minutinhos e vale bastante a pena, recomendo:

[First Impressions: The Value of the "Noob” - Emily Dunham (Ignite OSCON 2015)
](https://www.youtube.com/watch?v=zKWsBodjvC8)

**Comment by leoamrl** [comment]
Respondido.
